### PR TITLE
Moving chapters from remote collaboration to the main guide

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #

* This pull request is moving chapters from the remote collaboration outside

### List of changes proposed in this PR (pull-request)

* Move general chapters outside so that we focus on specific things in Remote and hybrid collab
* edit to make sure that they are general and don't mention remote collab only as originally intended

### What should a reviewer concentrate their feedback on?

- [ ] Changes made speak to general collab (rather than remote)
- [ ] Remote collab chapter has links to the chapters that have been moved out
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by bsipocz** [comment]
I've rebased this to make sure the warnings in CI are not causing any issues, and to pick up the more strict CI settings (a lot of infra fixes had been merged recently, so some if not all of those warnings should be ideally gone).

**Comment by bsipocz** [comment]
I'm sorry @malvikasharan that I picked this PR as a guinea pig for checking on the CI.

Hey infrapeople (@sgibson91, @da5nsy, @JimMadge, @aleesteele), here is a wonderful showcase that the recent pushes work no nicely. This PR got the all-green go-ahead earlier today as CI wasn't picking up on the broken links due to the renames/moving of resources. It may also highlight that it may be worth going into and rebase PRs once they are ready for merge (so we don't put the burden on the contributors, but fix them up quickly ourselves, before the regressions end  up in `main` and break any new PRs)

**Comment by bsipocz** [comment]
OK, so TOC changes have been done which should fix the CI errors. The order may still not be the perfect one, but I leave that for the content review.

**Comment by da5nsy** [comment]
@bsipocz 
> It may also highlight that it may be worth going into and rebase PRs
Would you be able to share your recommended workflow for doing this? It's not something I feel confident about doing

This might be a nice first test case for #2809 ?

**Comment by bsipocz** [comment]
The simples is to use the github cmd line tool `gh`, and then do a git rebase. I have to admit, it comes very much a second nature for me, but maybe it's not ideal for the workflow for here. For this particular PR:

`gh pr checkout 2843`
`git rebase alan-turing-institute/main`
 - Fix the conflicts as long as there any any, do continue with `git rebase --continue`
-  Once done, force push back. Be careful as it will overwrite the remote (sometimes I do a `git diff alan-turing-institute/main` to make sure all the things are in there from the PR before pushing back, but nothing unrelated)

`git push alan-turing-institute collab-refactor -f`

**Comment by JimMadge** [comment]
Regarding rebases, using merges might be better here as it doesn't rewrite history. That way a `git pull` should work without problems.

I feel to me like the contest between merges and rebases is something like

rebase - neat commit log, rewriting history (conflicts more likely)
merge - mess commit log, preserved history

**Comment by JimMadge** [comment]
Also, I there is a rule in GitHub called something like "Ensure branches are up to date" which essentially means if you want to merge into main, you must have merged/rebased any changes on main into your branch first.

It's a bit strict but handy to avoid bugs which are not present in `main` and `feature_branch` but which appear when the two are merged.

**Comment by bsipocz** [comment]
> Also, I there is a rule in GitHub called something like "Ensure branches are up to date" which essentially means if you want to merge into main, you must have merged/rebased any changes on main into your branch first.

I would very much advise against using that, it really is very annoying in practice, and it would be even more so in a project like the TTW that has a lot of different contributors, and parallel-running PRs.

**Comment by malvikasharan** [comment]
Thank you Infrastructure Co-leads 💪🏽 
I have moved out some files with some minor changes so that @SaranjeetKaur can reference them in her current drafts for hybrid collaboration. Please merge this when ready - these chapters are previously reviewed.

File: book/website/collaboration/remote-collab/remote-collab-prosandcons.md
Comment (Arielle-Bennett) [inline book/website/collaboration/remote-collab/remote-collab-prosandcons.md:5]: two "for"s where only one is needed

**Review by Arielle-Bennett — APPROVED** [review]
Minor typo otherwise good to go!

EDIT: oops looks like it's still failing some CI checks

File: book/website/collaboration/remote-collab/remote-collab-prosandcons.md
Comment (malvikasharan) [reply book/website/collaboration/remote-collab/remote-collab-prosandcons.md]: Good catch, just updated in the main file. Thank you.

**Comment by malvikasharan** [comment]
Thanks all once again. I am merging this so that Saranjeet can deemo her work in relation to these chapters.

**Pull request merged**
